tests/data/generator/t_protocol/custom_types.py

```diff
@@ -67,8 +67,8 @@ def encode(data_model_protobuf_object, data_model_object: "DataModel") -> None:
         data_model_protobuf_object.list_field.extend(data_model_object.list_field)
         data_model_protobuf_object.dict_field.update(data_model_object.dict_field)
 
-    @staticmethod
-    def decode(data_model_protobuf_object) -> "DataModel":
+    @classmethod
```

Comment:
this is a staticmethod in all other protocols; why the change?

Reply:
You can add it as an issue to change later but not now

Reply:
We also decided it to be a staticmethod a while back; so probably no issue either.